### ISSUE TITLE
How to train cifar10 using MOCO or other self-supervised learning?

### ISSUE THREAD

**Issue by saifhassan** [JegZheng/CACR-SSL#1]
Hey @JegZheng 

I implemented this repo but could not find any code to train any self-supervised method along with ResNet18 backbone on cifar10 dataset?

Thanks

**Comment by JegZheng** [comment]
Hi, we did not use ResNet18 or use MoCo to train models on cifar10 in our paper. We provide our code with AlexNet encoder to help reproduce results in our small-scale experiments.
However, you are welcome to do it with this repo by replacing the AlexNet encoder architecture with ResNet18.

**Comment by saifhassan** [comment]
> Hi, we did not use ResNet18 or use MoCo to train models on cifar10 in our paper. We provide our code with AlexNet encoder to help reproduce results in our small-scale experiments. However, you are welcome to do it with this repo by replacing the AlexNet encoder architecture with ResNet18.

yeah that's fine, i have trained with ResNet and other backboens, however I couldn't find any code which uses self supervised learning method such as MoCo on CIFAR10 dataset. Can you guide me in few steps, how would I train AlexNet or any backbone with MoCo or any other SSL method?

**Comment by JegZheng** [comment]
I think our small-scale experiments code is what you need. Please check that.

**Comment by saifhassan** [comment]
> I think our small-scale experiments code is what you need. Please check that.

Let me explain my query. Actually I want to apply self-supervised learning method on my dataset. I have gone through your small-experiments code but it doesn't use any self-supervised learning method (such as, MoCo, SimCLR or others). I want to use SSL along with CARD model.

My Questions:

1. Does your small experiments section use any SSL method on CIFAR10/CIFAR100 dataset?
2. If yes, then can you point it out the file please?
